src/api/index.js

```diff
@@ -56,11 +56,21 @@ const cancelMeeting = (meetingId, roomEmail) => agent
   .delete(`${apiBaseUrl}/room/${roomEmail}/meeting/${meetingId}`)
   .then(message => message)
 
+const addUser = (user, token) => agent
```

Comment:
The server side endpoint currently isn't protected but it's good to have the token there.